peer.go

```diff
@@ -1455,8 +1458,11 @@ func (p *peer) handleResponseClosingSigned(msg *lnwire.ClosingSigned,
 	// confirmation of the closing transaction, and mark the channel as
 	// such within the database (once it's confirmed").
 	notifier := p.server.cc.chainNotifier
+	p.wg.Add(1)
 	go waitForChanToClose(uint32(bestHeight), notifier, nil, chanPoint,
 		&closeTxid, func() {
+			defer p.wg.Done()
```

Comment:
@halseth thanks for bringing this up, hadn't realized that the callback may not be executed